crates/core/tedge/src/cli/certificate/renew.rs

```diff
@@ -39,11 +39,6 @@ impl RenewCertCmd {
         let key_path = &self.key_path;
         let id = certificate_cn(cert_path).await?;
 
-        // Remove only certificate
-        tokio::fs::remove_file(&self.cert_path)
-            .await
-            .map_err(|e| CertError::IoError(e).cert_context(self.cert_path.clone()))?;
-
```

Comment:
Why removed? (though I think removing the current cert is done too early, before other validations)

Reply:
The removal was done too early (as you said before other validations) and was also useless as the certificate content is in any case overridden.